src/style.css

```diff
@@ -1 +1,44 @@
-/* add styles here */
+body {
+  margin: 0;
+  background-color:white;
+  display: block;
+
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items:center;
+}
+
+.stars:first-child {
+  margin-top: 0;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(1),
+.stars--4 .stars__star:nth-child(2),
+.stars--4 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(4),
+.stars--5 .stars__star:nth-child(n){
+  background-image: url(images/star-active.svg);
+}
```

Comment:
don't repeat all possible child elements instead use formula (an + b)
you can read more about it [here](https://css-tricks.com/how-nth-child-works/)